core/eval.go

```diff
@@ -39,6 +40,8 @@ func evalPING(args []string) []byte {
 	return b
 }
 
+// Adds new entry for the specifid key-value pair
+// as per the sent expiry time.
```

Comment:
Here are some guidelines for writing good comments. https://tip.golang.org/doc/comment would recommend going through this and rewriting. Let's be more verbose.

Reply:
sure thanks for sharing. Rewrote the comments as suggested.